### PR TITLE
Moves microphysics submodule to NCAR repo and adds water-friendly aerosol emissions for microphysics

### PR DESCRIPTION
**Pull request by AndersJensen-NOAA**
Moves microphysics submodule to official NCAR repository (TEMPO - Thompson-Eidhammer Microphysics Parameterization for Operations)
Adds surface emissions for water-friendly aerosols based on the initial value from the lowest model level

Tested using 3km 18-h CONUS simulations and 3-km 120-h hurricane simulation.

### CODE REVIEW

**Review by jderrico-noaa — APPROVED** [review]

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,3 @@
-[submodule "src/core_atmosphere/physics/GSL_cloud_physics"]
-	path = src/core_atmosphere/physics/GSL_cloud_physics
-	url = https://github.com/AndersJensen-NOAA/GSL_cloud_physics.git
-	branch = develop
\ No newline at end of file
+[submodule "src/core_atmosphere/physics/TEMPO"]
+	path = src/core_atmosphere/physics/TEMPO
+	url = https://github.com/NCAR/TEMPO.git
```
Comment (tanyasmirnova) [inline .gitmodules:3]: @AndersJensen-NOAA Anders, why are you changing GSL_cloud_physics to TEMPO? It seems to me less descriptive about its content and about the developer.

**Review by tanyasmirnova — APPROVED** [review]
@jderrico-noaa Janet, Anders's PR also contains my change to RUC LSM (module_sf_ruclsm.F) but not to VEGPARM.TBL. We should not lose the VEGPARM.TBL modification when we merge this PR. Thank you!

**Review by mdtoyNOAA — APPROVED** [review]

File: .gitmodules (hunk 1 above)
Comment (AndersJensen-NOAA) [reply .gitmodules:3]: @tanyasmirnova EMC wants to collaborate on microphysics development, so (after lots of discussion) we decided to move the submodule to an NCAR (neutral) repository and not keep it GSL specific.

**Comment by AndersJensen-NOAA** [comment]
> @jderrico-noaa Janet, Anders's PR also contains my change to RUC LSM (module_sf_ruclsm.F) but not to VEGPARM.TBL. We should not lose the VEGPARM.TBL modification when we merge this PR. Thank you!

@tanyasmirnova I can pull your changes of VEGPARM.TBL into my PR.

File: .gitmodules (hunk 1 above)
Comment (tanyasmirnova) [reply .gitmodules:3]: @AndersJensen-NOAA Thank you for explaining.

**Pull request merged**